Assets/Scripts/UI/Menu.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace OperationBlackwell.UI {
+	public class Menu : MonoBehaviour {
+
+		protected GameObject canvas_;
```

Comment:
Is it not easier to make these exposed to the editor and drag the correct objects to it there?

Reply:
I might've been slightly retarded here, but yes, likely.

Reply:
Are you going to change this, or is it merge time?

Reply:
Changing it right now.